packages/quantic/cypress/integration/standalone-search-box/standalone-search-box.cypress.ts

```diff
@@ -0,0 +1,92 @@
+import {configure} from '../../page-objects/configurator';
+import {interceptSearch} from '../../page-objects/search';
+import {StandaloneSearchBoxExpectations as Expect} from './standalone-search-box-expectations';
+import {StandaloneSearchBoxActions as Actions} from './standalone-search-box-actions';
+import {
+  clearLocalStorage,
+  typeQueriesListLocalStorage,
+} from '../../page-objects/actions/action-set-local-storage';
+import {scope} from '../../reporters/detailed-collector';
+
+interface StandaloneSearchBoxOptions {
+  placeholder: string;
+  withoutSubmitButton: boolean;
+  numberOfSuggestions: number;
+  redirectUrl: string;
+}
+
+describe('quantic-standalone-search-box', () => {
+  const standaloneSearchBoxUrl = 's/quantic-standalone-search-box';
+
+  function visitStandaloneSearchBox(
+    options: Partial<StandaloneSearchBoxOptions> = {}
+  ) {
+    interceptSearch();
+    cy.visit(standaloneSearchBoxUrl);
+    configure(options);
+  }
+
+  describe('with default option', () => {
```

Comment:
The main feature of the standalone searchbox is that it redirects to a search page with a given query. So I think we should be testing with option `redirectUrl="/full-search-example"` and test that after the redirect is made to the example search page the correct query is made whether it was a typed queries or a query suggestion selection.

Reply:
I tested it with both url and it went well